tests/ts_simple_create_table_eqc.erl

```diff
@@ -0,0 +1,53 @@
+%% -*- Mode: Erlang -*-
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_simple_create_table_eqc).
+-compile(export_all).
+
+-behavior(riak_test).
+
+%-ifdef(EQC).
```

Comment:
We'll definitely want to conditionally include the EQC headers because if you don't have QuickCheck installed, the compilation of `riak_test` will fail